runtime/src/main/java/org/corfudb/runtime/collections/Query.java

```diff
@@ -163,6 +163,20 @@ public int count(@Nonnull final String tableName,
         }
     }
 
+    @Nonnull
+    private <K extends Message, V extends Message, M extends Message>
+    List<CorfuStoreEntry<K, V, M>> scanAndFilterByEntry(
```

Comment:
Do we need scanAndFilter() above? Can we just have this method only?

Reply:
Done.